### PR TITLE
feat(cd): ghcr.io setup and docker image build workflow

### PR DESCRIPTION
**Pull request by kshychko**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [x] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [vc-kit doc site](https://uncefact.github.io/vckit/)
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by ashleythedeveloper** [comment]
@namhoang1604, any update on reviewing this PR?

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-#      - cd_*
-#    tags:
-#      - "v[0-9]+.[0-9]+.[0-9]+"
-#  pull_request:
+  push:
+    branches:
+      - next
+      - cd_*
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```
Comment (namhoang1604) [inline .github/workflows/package.yml:9]: I think the workflow is only triggered when creating a tag

Comment (kshychko) [reply .github/workflows/package.yml]: @namhoang1604 , I removed branches but left workflow dispatch option. it is handy when a workflow can be ran manually.

**Review by namhoang1604 — APPROVED** [review]

**Comment by namhoang1604** [comment]
LGTM

**Pull request merged**